src/index.html

```diff
@@ -4,10 +4,242 @@
+        </a>
+        <a
+          href="tel:+1 234 555-55-55"
+          class="header__phone-link"
+        >
+        </a>
+      </div>
+      <h1 class="header__title">Miami<br>Condo Kings</h1>
+    </header>
+    <nav class="nav">
+      <a href="#" class="nav__link">Home</a>
+      <a href="#about" class="nav__link">About us</a>
+      <a href="#gallery" class="nav__link">Gallery</a>
+      <a href="#testimonials" class="nav__link">Testimonials</a>
+      <a href="#contact" class="nav__link">Contact us</a>
+      <a href="tel:+1234555-55-55" class="nav__link">+1 234 555-55-55<br> <br> Call to order</a>
+    </nav>
+    <main class="main">
+      <article class="about" id="about">
+        <h2 class="about__title">
+          Miami best <br>
+          Real estate
+          <a href="#" class="about__accent">here</a>
+        </h2>
+        <section class="about__discription">
+          <h3 class="about__subtitle">
+            Over 50 years experience in love.
+          </h3>
+          <section class="about__text">
+            <p class="about__paragraph">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p class="about__paragraph">
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <strong class="about__paragraph">
+            We will never be late!<br>
+              Guaranteed secure transaction - 100%
+            </strong>
+          </section>
+        </section>
+      </article>
+      <article class="gallery" id="gallery">
+        <section class="gallery__block">
+          <h2 class="gallery__title">Gallery</h2>
+          <img src="./images/crown.svg" alt="Crown">
+        </section>
+        <section class="gallery__block">
+          <div class="gallery__images-wrapper gallery__images-wrapper--1">
+            <img
+              src="./images/photos/1.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+            <img
+              src="./images/photos/2.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+          </div>
+          <h3 class="gallery__subtitle">
+            <a
+              class="gallery__subtitle-link"
+              href="https://goo.gl/maps/vYt2Ckb1FUw2ZeVk8"
+            >
+              Marina Palms
+            </a>
+            / North Miami Beach, FL 33162
+          </h3>
+        </section>
+        <section class="gallery__block">
+          <div class="gallery__images-wrapper gallery__images-wrapper--2">
```

Comment:
```suggestion
          <div class="gallery__images-wrapper">
```
You can remove `--2` because `gallery__images-wrapper` has 1fr 1fr itself